lib/npg_qc/autoqc/qc_store.pm

```diff
@@ -191,7 +191,7 @@ sub run_from_db {
     }
     return $c;
   }
-  foreach my $check_name (@{npg_qc::autoqc::autoqc->checks_list()}) {
+  foreach my $check_name (grep {!/rna_seqc/sm} @{npg_qc::autoqc::autoqc->checks_list()}) {
```

Comment:
Hardcoding was OK for a prototype. For production code we probably should deturmine dynamically whether the database representation exists.